hyperopt/fmin.py

```diff
@@ -381,6 +387,10 @@ def fmin(
         Limits search time by parametrized number of seconds.
         If None, then the search process has no time constraint.
 
+    confidence : None or double, default None
+        Limits search time when minimal loss reduced to certain amount.
+        If None, then the search process has no best_loss constraint.
```

Comment:
I think the term confidence is used for other concepts in ML/maths, so I would go with "tolerance", "loss_threshold", "loss_cutoff", or similar. About the docstring, what do you think about

```Limits search when loss reaches at least the max_loss/tolerance/whatever value.
        If None, then the search process has no constraint on the loss value.```

Reply:
I would prefer loss_threshold, will change it soon

Reply:
in the first sentence you talk about "minimal loss" while in the first one about the "best_loss". I think it's a bit confusing. What about something like:

```Limits search time when the loss for the best trial is lower or equal than this value.
        If None, then the search process has no constraint on the loss, and will stop
        based on other parameters, e.g. `max_evals`, `timeout`.```